transports/daily/src/transport.ts

```diff
@@ -274,6 +367,17 @@ export class DailyTransport extends Transport {
     });
   }
 
+  private stopRecording() {
+    if (this._bufferLocalAudioUntilBotReady) {
```

Comment:
should this look more like the check in `disconnect` so that we don't have someone flip this flag and then the recorder keeps going?

```suggestion
    if (this.mediaStreamRecorder && this.mediaStreamRecorder.getStatus() !== "ended")
```

Reply:
Done.